cluster-scope/overlays/nerc-ocp-prod/oauths/cluster_patch.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+    - mappingMethod: claim
+      name: mss-keycloak
+      openID:
+        claims:
+          email:
+            - email
+          name:
+            - name
+          preferredUsername:
+            - preferred_username
+        clientID: ocp-prod
```

Comment:
just wanted to make sure this is what we are calling it instead of nerc-ocp-prod

Reply:
[Correct](https://github.com/nerc-project/mss-keycloak/blob/ca40cfde16e44ecc60a00f369ce9bbaa666448eb/k8s/overlays/nerc-shift-1/clients/ocp-prod.yaml#L37), the client for openstack was just named openstack, and nerc seems reduntant.